src/richie/plugins/lti_consumer/cms_plugins.py

```diff
@@ -0,0 +1,60 @@
+"""
+LTI consumer CMS plugin
+"""
+import json
+
+from django.utils.translation import gettext_lazy as _
+
+from cms.plugin_base import CMSPluginBase
+from cms.plugin_pool import plugin_pool
+
+from richie.apps.core.defaults import PLUGINS_GROUP
+
+from .forms import LtiConsumerForm
+from .models import LtiConsumer
+
+
+@plugin_pool.register_plugin
+class LtiConsumerPlugin(CMSPluginBase):
```

Comment:
I think in marsha we chose to write LTI in capital letters, so maybe keep this pattern :man_shrugging:  `LTIConsumerPlugin`

Reply:
The problem with LTI (uppercase) is at the call for react component :
with LTI, we have to write `class="richie-react richie-react--l-t-i-consumer"` which I found awful…

Reply:
So I choose to use LtiConsumer everywhere.

Reply:
I didn't get how the Python class name is link to the class attribute.

Reply:
Oh, there's no direct link, but I thought having the same name everywhere were better for consistency.

Reply:
This comment could be also applied for frontend

Reply:
I mean : having the same name for the django plugin/model and the react component.